### PR TITLE
create Create Account Failed page

### PR DESCRIPTION
**Pull request by hyunhee804**
Created a page when creating an account failed.

Will implement and connect more when I finish creating successful page of create account

### CODE REVIEW

File: Test2/app/src/main/res/layout/activity_create_acc_fail.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".CreateAccFail">
+
+    <ImageView
+        android:id="@+id/imageView4"
```
Comment (kimhajiny) [inline Test2/app/src/main/res/layout/activity_create_acc_fail.xml:10]: Next time we would need more specific ids for all attributes

File: Test2/app/src/main/res/layout/activity_create_acc_fail.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        android:layout_height="154dp"
+        android:layout_marginTop="172dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.497"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/t_pay_image" />
+
+    <TextView
+        android:id="@+id/textView9"
```
Comment (kimhajiny) [inline Test2/app/src/main/res/layout/activity_create_acc_fail.xml:21]: Here too

**Review by kimhajiny — APPROVED** [review]

File: Test2/app/src/main/res/layout/activity_create_acc_fail.xml (hunk 1 above)
Comment (hyunhee804) [reply Test2/app/src/main/res/layout/activity_create_acc_fail.xml:10]: Yup! I keep forgetting about them. I will change all the attribute ids in a few pr

Comment (Chaewon382) [reply Test2/app/src/main/res/layout/activity_create_acc_fail.xml:10]: Agreed and thanks Hyunhee!

**Pull request merged**